src/olympia/promoted/admin.py

```diff
@@ -81,3 +109,9 @@ def formfield_for_foreignkey(self, db_field, request=None, **kwargs):
             return SlugOrPkChoiceField(**kwargs)
         return super().formfield_for_foreignkey(
             db_field, request, **kwargs)
+
+    def has_delete_permission(self, request, obj=None):
+        qs = PrimaryHeroInline.model.objects.filter(enabled=True)
```

Comment:
Just wondering, why is this using `PrimaryHeroInline.model.objects` rather than just `PrimaryHero.objects`?

Reply:
I thought not having to import the `PrimaryHero` model was better, but I dunno?

Reply:
I dunno either. I was just asking. If there is no difference then I guess just leave it as is.